source/develop/release-management/features/metrics/metrics-store-installation.html.md

```diff
@@ -39,66 +39,28 @@ In oVirt 4.2 there will be an option to add SSO: [Metrics Store setup on top of
 
 
 [![oVirt Metrics data flow](/images/wiki/oVirtMetricsDataFlow.jpg)](images/wiki/oVirtMetricsDataFlow.jpg)
+
 Once you have finished this step, you should have:
```

Comment:
Change to:
"Once you have finished this step you should have:"

Reply:
Done